packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx

```diff
@@ -185,4 +186,14 @@ export class CalendarMonth extends BaseComponent<ICalendarMonthProps, {}> {
       onHeaderSelect(true);
     }
   }
+
+  private _formatYear(navigatedDate: Date) {
+    let { formatDate } = this.props;
+    let formatedDate = navigatedDate.getFullYear().toString();
+
+    if (formatDate && formatDate.formatYear) {
+      formatedDate = formatDate.formatYear(navigatedDate);
+    }
+    return formatedDate;
+  }
```

Comment:
Same comment here, you could create one object that is the default with all 3 functions defined